client/src/App.js

```diff
@@ -109,7 +109,7 @@ function App() {
                  src="https://user-images.githubusercontent.com/13723156/127782418-a009e7a2-294d-4abe-b46e-2ac0baac5e40.png" 
                  alt="Circle logo"  
                />
-               <LinearGradient height="200%" /> 
+               <LinearGradient height="200%" width= "4px" /> 
```

Comment:
I think linear gradient should automatically apply width if height is set, and vice versa. Can you test out that this component works fine still when you drop the `width` property?

Reply:
Yep I checked and it worked. However when we drop the width/height property depending on the situation, the line becomes thinner; that is the only difference